sms_logs/src/alerts.test.ts

```diff
@@ -0,0 +1,106 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
```

Comment:
question: We don't seem to have an explicit `any` in this file. What is this needed for?

Reply:
Probably just left over from some changes